docs/contributing/writing-instrumentation.md

```diff
@@ -384,3 +384,18 @@ dependencies {
   testImplementation(project(":instrumentation:yarpc-1.0:javaagent"))
 }
 ```
+
+## Why we don't use ByteBuddy @Advice.Origin Method
```

Comment:
WDYT about moving it to the `writing-instrumentation-module` file, just below the `Associate instrumentation classes with instrumented library classes` section? It seems like it'd fit that doc more than this one (since it already describes details of writing an advice class)